LnkParse3/decorators.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+from datetime import timezone
+from struct import unpack
+import functools
+from LnkParse3.lnk_exception import lnk_exception
+
+
+def must_be(expected):
+    def outer(func):
+        @functools.wraps(func)
+        def inner(self, *args, **kwargs):
+            result = func(self, *args, **kwargs)
+
+            # FIXME: delete
+            return result
```

Comment:
I would just return the result in the case of `force`. Also, some warning can be logged into the stdout.

Reply:
OK, I will make `must_be` decorator to warn something in another P-r.